gpu-enabler/src/main/scala/com/ibm/gpuenabler/ColumnPartitionSchema.scala

```diff
@@ -21,12 +21,20 @@ import java.io.{ObjectInputStream, ObjectOutputStream}
 
 import org.apache.spark.gpuenabler.CUDAUtils
 import org.apache.spark.util.Utils
+import org.apache.spark.gpuenabler.CUDAUtils.sparkUtils.tryOrIOException
```

Comment:
While I am not sure whether all of new imports are necessary, could you double-check whether they are necessary?
(e.g. scala.reflect.ClassTag is imported twice).